src/test/lib/statistics/attribute_statistics_test.cpp

```diff
@@ -65,6 +65,32 @@ TEST_F(AttributeStatisticsTest, Scaled) {
   EXPECT_EQ(scaled_attribute_statistics->distinct_value_count->count, 123);
 }
 
+// Scalin with a selectivity of 1 does noch generate a new object.
+TEST_F(AttributeStatisticsTest, ScaledWithIdentity) {
+  const auto attribute_statistics = std::make_shared<AttributeStatistics<int32_t>>();
+  const auto histogram = GenericHistogram<int32_t>::with_single_bin(0, 100, 40, 20);
+  const auto min_max_filter = std::make_shared<MinMaxFilter<int32_t>>(0, 100);
+  const auto range_filter = std::make_shared<RangeFilter<int32_t>>(std::vector{std::pair<int32_t, int32_t>(0, 100)});
+  const auto null_value_ratio = std::make_shared<NullValueRatioStatistics>(0.2f);
+  const auto distinct_value_count = std::make_shared<DistinctValueCount>(123);
```

Comment:
On purpose with a `DistinctValueCount` statistic of 123 while only having 40 values?

Reply:
no, just copy/paste